src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/WhatIsTheNewStartDateViewModelMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers.Apprentice
+{
+    public class WhatIsTheNewStartDateViewModelMapper : IMapper<EmployerLedChangeOfProviderRequest, WhatIsTheNewStartDateViewModel>
+    {
+
+        private readonly ICommitmentsApiClient _client;
+
+        public WhatIsTheNewStartDateViewModelMapper(ICommitmentsApiClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<WhatIsTheNewStartDateViewModel> Map(EmployerLedChangeOfProviderRequest source)
+        {
+            var apprenticeship = await _client.GetApprenticeship(source.ApprenticeshipId);
+            
+            return new WhatIsTheNewStartDateViewModel
```

Comment:
this may need mapping for enddate & price ( when coming from the confirmation page).

Reply:
This will be resolved as part of the Check your answers page CON-2521